packages/dialtone-vue2/recipes/conversation_view/wysiwyg_editor/wysiwyg_editor.vue

```diff
@@ -0,0 +1,574 @@
+      show-footer
+      kind="default"
+      size="default"
+      :close-button-props="{ ariaLabel: 'Close the dialog' }"
+      @update:show="(s) => updateInputModal(s)"
+    >
+      <div class="d-m4">
+        <dt-rich-text-editor
+          v-model="linkInput"
+          :input-aria-label="showAddLink.setLinkInputAriaLabel"
+          data-qa="dt-wysiwyg-editor-link-input"
+          :link="true"
+          :output-format="textOutputFormat"
+          :placeholder="setLinkPlaceholder"
+          input-class="d-bgc-black-100 d-bar4 d-ba d-baw1 d-bc-black-300 d-pl6 d-py4 d-ol-none"
+          @click="onInputFocus($event)"
+          @click.native.stop="onInputFocus($event)"
+          @focus="onInputFocus($event)"
+          @enter.native="setLink"
+        />
+      </div>
+      <template #footer>
+        <dt-button
+          :aria-label="confirmSetLinkButton.ariaLabel"
+          data-qa="dt-wysiwyg-editor-set-link-confirm-btn"
+          @click="setLink"
+        >
+          {{ confirmSetLinkButton.label }}
+        </dt-button>
+        <dt-button
+          :aria-label="cancelSetLinkButton.ariaLabel"
+          importance="clear"
+          kind="muted"
+          data-qa="dt-wysiwyg-editor-set-link-cancel-btn"
+          @click="closeLinkInputModal"
+        >
+          {{ cancelSetLinkButton.label }}
+        </dt-button>
+        <dt-button
+          :aria-label="removeLinkButton.ariaLabel"
+          importance="clear"
+          kind="muted"
+          data-qa="dt-wysiwyg-editor-remove-link-btn"
+          @click="removeLink"
+        >
+          {{ removeLinkButton.label }}
+        </dt-button>
+      </template>
+    </dt-modal>
+
+    <!-- Some wrapper to restrict the height and show the scrollbar -->
+    <div
+      class="d-of-auto d-mx16 d-mt8 d-mb16 d-c-text"
+      :style="{ 'max-height': maxHeight }"
+    >
+      <dt-rich-text-editor
+        ref="richTextEditor"
+        v-model="internalInputValue"
+        data-qa="dt-rich-text-editor"
+        :editable="editable"
+        :input-aria-label="inputAriaLabel"
+        :input-class="inputClass"
+        :output-format="htmlOutputFormat"
+        :auto-focus="autoFocus"
+        :link="linkOptions"
+        :placeholder="placeholder"
+        :allow-line-breaks="true"
+        v-bind="$attrs"
+        @focus="onFocus"
+        @blur="onBlur"
+        @input="onInput($event)"
+      />
+    </div>
+  </div>
+</template>
+
+<script>
+/* eslint-disable max-lines */
+import {
+  DtRichTextEditor,
+  RICH_TEXT_EDITOR_OUTPUT_FORMATS,
+  RICH_TEXT_EDITOR_AUTOFOCUS_TYPES,
+} from '@/components/rich_text_editor';
+import {
+  WYSIWYG_EDITOR_SUPPORTED_LINK_PROTOCOLS,
+  WYSIWYG_EDITOR_DEFAULT_LINK_PREFIX,
+} from './wysiwyg_editor_constants.js';
+import { DtIcon } from '@/components/icon';
+import { DtButton } from '@/components/button';
+import { DtModal } from '@/components/modal';
+
+export default {
+  name: 'DtRecipeWysiwygEditor',
```

Comment:
I'm a bit reluctant to call it WYSIWYG because some people might not know what that means.. Searched around a bit in other component libraries to see what they call it...

Quasar just calls it "Editor" https://quasar.dev/vue-components/editor/
Lightning calls it "Rich Text Editor" https://www.lightningdesignsystem.com/components/rich-text-editor/
We probably can't call it that because of our existing rich-text-editor component... unless we renamed that one to rich-text-editor-base
Atlassian calls it "Editor" as well https://atlaskit.atlassian.com/packages/editor/editor-core
MetLife does in fact call it WYSIWYG, but then also has "rich text editor" in brackets https://design.metlife.com/resources/responsive-web/components/content-container/wysiwyg/

hmmmm....

Reply:
Yes, I completely agree, also it's a bit hard to type in every time bc it's a rather long acronym 😅 I think we can go with Editor for now and if we get a better idea later we can change it again.